src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -28,6 +30,71 @@ public SarifValidatorTests()
             _schema = SchemaReader.ReadSchema(schemaText, JsonSchemaFile);
         }
 
+        //[Fact]
+        internal void ValidateAllTheThings()
+        {
+            // First, we start with builders that only populate required properties that are backed by primitives.
+            IDictionary<Type, DefaultObjectPopulatingVisitor.PrimitiveValueBuilder> propertyValueBuilders =
+                DefaultObjectPopulatingVisitor.GetBuildersForAllPrimitives();
```

Comment:
>GetBuildersForAllPrimitives [](start = 47, length = 27)

Should this have been `ForRequiredPrimitives`? Either this or the comment seems wrong (I think the comment is wrong, given the test name).